### PR TITLE
Logic to not rsync if build failed

### PR DESCRIPTION
**Pull request by CorneliusIV**
This connects #23

### CODE REVIEW

File: api.go [hunk 1]
```diff
@@ -154,19 +154,26 @@ func GrabBuiltStaticFiles(dockerImageID, projectName, transferLocation string) {
...
+			logging.LogToFile("There was an error building the docker container")
+			makePutRequest(dockerInfo, "FAL")
+			return "fail"
 		case buildTag := <-c1:
 			logging.LogToFile("Container built...transfering built files...")
-			GrabBuiltStaticFiles(buildTag, projectName, buildDir)
+			GrabBuiltStaticFiles(buildTag, dockerInfo.REPO_NAME, buildDir)
 			if Config.ENV != "test" {
-				rsyncProject(buildDir+"/public/*", Config.DEPLOYROOTPATH+projectPath)
+				rsyncProject(buildDir+"/public/*", dockerInfo.PATH)
```
Comment (oudeismetis) [inline api.go]: this line was added recently because we have to rsync to a specific remote folder. Lets discuss.

Comment (CorneliusIV) [inline api.go]: actually this was more of typo good catch

**Comment by CorneliusIV** [comment]
@oudeismetis fixed

**Comment by oudeismetis** [comment]
:+1:

**Comment by JulianGindi** [comment]
This looks very solid :+1:

**Comment by CorneliusIV** [comment]
Can I pull this?

**Comment by oudeismetis** [comment]
yes yes. do it!

**Comment by recursivefunk** [comment]
be careful, it my fall if you pull too hard

**Pull request merged**